
float32 head_x
float32 head_y
float32 head_z
float32 head_yaw
float32 head_pitch

float32 neck_x
float32 neck_y
float32 neck_z

float32 torso_x
float32 torso_y
float32 torso_z

float32 left_shoulder_x
float32 left_shoulder_y
float32 left_shoulder_z

float32 left_elbow_x
float32 left_elbow_y
float32 left_elbow_z

float32 left_hand_x
float32 left_hand_y
float32 left_hand_z

float32 left_hip_x
float32 left_hip_y
float32 left_hip_z

float32 left_knee_x
float32 left_knee_y
float32 left_knee_z

float32 left_foot_x
float32 left_foot_y
float32 left_foot_z

float32 right_shoulder_x
float32 right_shoulder_y
float32 right_shoulder_z

float32 right_elbow_x
float32 right_elbow_y
float32 right_elbow_z

float32 right_hand_x
float32 right_hand_y
float32 right_hand_z

float32 right_hip_x
float32 right_hip_y
float32 right_hip_z

float32 right_knee_x
float32 right_knee_y
float32 right_knee_z

float32 right_foot_x
float32 right_foot_y
float32 right_foot_z


